ToopherSecondFactor/toopher-openam.js

```diff
@@ -1,183 +1,84 @@
 // vim: ts=2 sw=2 expandtab cindent
```

Comment:
This _does_ seem like something that should live in your .vimrc file instead of the source.  Any strong reason you can't do that to get the same effect without injecting editor/developer specific stuff into the code?